helm-charts/konk/scripts/provision.sh

```diff
@@ -29,3 +29,10 @@ then
     --from-file=/etc/kubernetes/admin.conf
   kubectl -n $NAMESPACE label secret $FULLNAME-kubeconfig $LABELS
 fi
+
+sleep 1m
+
+DEPLOYMENT_UID=$(kubectl get deploy -n $NAMESPACE $FULLNAME -o yaml | grep uid | cut -c 8-)
+kubectl patch -n $NAMESPACE secret $FULLNAME-apiserver-cert -p '{"metadata":{"ownerReferences":[{"apiVersion":"v1", "kind":"Deployment", "name":"'${FULLNAME}'", "uid":"'${DEPLOYMENT_UID}'"}]}}'
```

Comment:
Not `apps/v1`? I'm surprised this works.
```suggestion
kubectl patch -n $NAMESPACE secret $FULLNAME-apiserver-cert -p '{"metadata":{"ownerReferences":[{"apiVersion":"apps/v1", "kind":"Deployment", "name":"'${FULLNAME}'", "uid":"'${DEPLOYMENT_UID}'"}]}}'
```

Reply:
That's a good point. Not sure how that worked, but I replaced the apiVersion with `apps/v1` to be on the safe side.